Blazorade.Bootstrap.Components/ProgressBar.razor.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Blazorade.Bootstrap.Components
+{
+    public partial class ProgressBar
+    {
+        /// <summary>
+        /// If set to true, this progress bar will have a striped texture.
+        /// </summary>
+        [Parameter]
+        public bool IsStriped { get; set; }
+
+        /// <summary>
+        /// If set to true, any stripe will be animated.
+        /// </summary>
+        [Parameter]
+        public bool IsAnimated { get; set; }
+
+        /// <summary>
+        /// The minimum value represented by the progress bar. This must be less than both Value and ValueMax.
+        /// </summary>
+        [Parameter]
+        public double ValueMin { get; set; }
+
+        /// <summary>
+        /// The maximum value represented by the progress bar. This must be greater than both ValueMin and Value.
+        /// </summary>
+        [Parameter]
+        public double ValueMax { get; set; }
```

Comment:
In other components, Min/Max is used as prefix. For instance [CarouselItem](https://github.com/Blazorade/Blazorade-Bootstrap/blob/master/Blazorade.Bootstrap.Components/CarouselItem.razor.cs#L47).

So, rather MinValue and MaxValue.